test-tools/issuer-front-end/src/constants.ts

```diff
@@ -1,7 +1,7 @@
 import { BrowserWalletConnector, ephemeralConnectorType } from '@concordium/react-components';
 import moment from 'moment';
 
-export const EXAMPLE_CREDENTIAL_SCHEMA = `https://raw.githubusercontent.com/Concordium/concordium-web3id/ac803895b1ffaa50888cfc6667331f6ebb0b889e/examples/json-schemas/education-certificate/JsonSchema2023-education-certificate.json`;
+export const EXAMPLE_CREDENTIAL_SCHEMA = 'https://raw.githubusercontent.com/Concordium/concordium-web3id/main/examples/json-schemas/education-certificate/JsonSchema2023-education-certificate.json';
```

Comment:
That is not a good idea to point this to `main` instead of a fixed commit in my opinion. I had this initially but then the `JSONSchema2023-education-certificate.json` got updated on the `main` branch and that broke this dependent front end. People updating the schema in the `main` branch will not be aware that this project depends on it. Can we use a fixed commit please.

Reply:
Yes, I actually intended to point to the commit that is currently `main`. Will fix.